Dockerfile

```diff
@@ -17,13 +17,33 @@
 FROM busybox
 LABEL maintainer="Chef Software, Inc. <docker@chef.io>"
 
-ARG CHANNEL=stable
-ARG VERSION=18.3.0
+ARG CHANNEL=unstable
+ARG VERSION=19.0.49
```

Comment:
As long as this workflow here still works this will be fine:

https://github.com/chef/chef/blob/ashiqueps/chef-hab-docker-image-updates/.expeditor/update_dockerfile.sh

Reply:
Got it.